src/transformers/tokenization_utils_base.py

```diff
@@ -700,8 +700,11 @@ def convert_to_tensors(
                 raise ImportError("Unable to convert output to PyTorch tensors format, PyTorch is not installed.")
             import torch
 
-            as_tensor = torch.tensor
             is_tensor = torch.is_tensor
+
+            def as_tensor(value, dtype=None):
+                return torch.tensor(np.array(value))
+
```

Comment:
Do we also gave to handle case of a ragged lists as it is done with jax? Also wondering if this should always be done or not.  In this case we have a ` list of numpy.ndarrays ` but what if we have a ` numpy.ndarrays ` or a list of list, does adding a conversion have an impact? 
It might be a total nit, but I'm just wondering if we should just add a check to do it

Reply:
I see.  but i'm not quit sure it's easy to do.

Reply:
@sgugger 

As the previous code is just `as_tensor = torch.tensor`, we don't need to worry about the different input types `value` here.

The only thing I don't like much is it now takes an extra line to see we are doing is `torch.tensor`. But as it indeed make things faster (~30x), I am good.

Is this PR good for you?